akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardRegion.scala

```diff
@@ -737,12 +750,20 @@ private[akka] class ShardRegion(
     }
   }
 
+  def queryCoordinator(query: ShardingQuery, to: ActorRef): Unit =
+    coordinator.fold(to ! ClusterShardingInitializing)(_.tell(query, to))
```

Comment:
`foreach` rather than `fold` given that it does not return anything

Reply:
`foreach` wouldn't do anything in the `None` case, though?

What was wrong about `forward`, btw?

Reply:
Ah, I missed the response for initializing part. NVM!